docs/glome-login.md

```diff
@@ -83,53 +71,62 @@ server to detect truncated requests and reject those early. Without the
 trailing slash requirement the request will still likely look correct but may
 result in an invalid request being signed causing confusion for the operator.
 
+#### Host ID
+
+The client identifies itself as a named host, using the host id field. This id
```

Comment:
I'd have a small nitpicky preference of s/id/ID/ because it's an abbreviation (and that also makes it consistent with the heading).